src/app/store.ts

```diff
@@ -1,10 +1,18 @@
 import { configureStore, ThunkAction, Action } from '@reduxjs/toolkit';
 // eslint-disable-next-line import/no-cycle
```

Comment:
Consider removing the commented line with import/no-cycle eslint disable, as it's not needed anymore.